webapp/app.py

```diff
@@ -166,3 +178,63 @@ def get_jira_tasks(webpage_id: int):
         return jsonify(tasks), 200
     else:
         return jsonify({"error": "Failed to fetch Jira tasks"}), 500
+
+
+@app.route("/api/create-page", methods=["POST"])
+@login_required
+def create_page():
+    data = request.get_json()
+
+    project = data.get("project")
+    name = data.get("name")
+    copy_doc = data.get("copy_doc")
+    owner = data.get("owner")
+    reviewers = data.get("reviewers")
+    parent = data.get("parent")
```

Comment:
We should add these to a schema to get automatic validation, and also autoloading of the values